src/lib.rs

```diff
@@ -51,6 +54,9 @@ pub struct AppState {
 }
 
 pub static COOKIES_SECRET: OnceCell<Key> = OnceCell::new();
+pub static EMAIL_USERNAME: OnceCell<String> = OnceCell::new();
+pub static EMAIL_PASSWORD: OnceCell<String> = OnceCell::new();
+pub static EMAIL_SMTP_SERVER: OnceCell<String> = OnceCell::new();
```

Comment:
the secret key was declared in this way because it can't be passed in the `AppState`, but you can add the rest in the `AppState` struct rather than using `OnceCell`

Reply:
why can't the secret key by passed in `AppState` but these can?

Reply:
The type is not send or something, forgot